source/how-tos/debug/debug-apache.rst

```diff
@@ -77,7 +79,39 @@ Or you're using the wrong hostname in your browser.
 
           sudo /sbin/apache2 -S
 
+Performance Tuning
+------------------
+
+If you're servicing many clients at time (more than 50) you will likely need to change the
+`Apache Httpd's worker configuration`_. The default configuration may degrade service when
+Httpd has to serve many clients (I.e., when you have a lot of customers using Open OnDemand).
+
+We suggest configurations similar to this. 
+
+.. warning:: 
+  `ServerLimit` and `ThreadsPerChild` should not exceed the number of cores available. 8
```

Comment:
Per other comment, threads per child can very easily exceed number of cores as you typically can have a lot more threads on a host than number of CPUs.